assets/base.css

```diff
@@ -3120,3 +3120,60 @@ details-disclosure > details {
 .rte blockquote > * {
   margin: -0.5rem 0 -0.5rem 0;
 }
+
+/* Animations */
+
+.scroll-animation__trigger {
+  transition: opacity 0.55s ease-in-out;
+  opacity: 0;
+}
+
+.scroll-animation__trigger--active {
+  animation: 0.55s ease-in-out slideIn forwards;
+}
```

Comment:
Do we need generic rules for `.scroll-animation__trigger` and `.scroll-animation__trigger--active` when there are specific rules below? I suggest _only_ using the specific rules on a section-by-section basis. I feel like `scroll-animation__trigger--active` shouldn't have any styles on its own.

Reply:
I think the Image banner is the only one that has specific styles, and these base classes cover everything else. Will we have more specific styles, @kjellr?

Reply:
Yeah, most of our animations are shared across sections, so I think keeping this rule generic makes sense. The [Image Banner](https://github.com/Shopify/dawn/blob/8fcc99a8c31a7b5afb9189a607ad25643ff0fde2/assets/base.css#L3092-L3107) and [Slideshow](https://github.com/Shopify/dawn/blob/8fcc99a8c31a7b5afb9189a607ad25643ff0fde2/assets/base.css#L3109-L3111) are the only ones that I assigned specific styles to in my demo.